beta/tests/tensorflow/sparsity/test_common.py

```diff
@@ -176,19 +176,19 @@ def test_polynomial_schedule_per_epoch_step(self,
 
         # After epoch 2
         self.run_epoch(steps_per_epoch, scheduler, mock, ref_sparsity_levels, explicit, epoch)
-        assert scheduler.current_sparsity_level == ref_sparsity_levels[3]
+        assert scheduler.sparsity_level == ref_sparsity_levels[3]
         epoch += 1
 
         # After epoch 3 - sparsity freeze should occur
         self.run_epoch(steps_per_epoch, scheduler, mock, ref_sparsity_levels, explicit, epoch)
         magnitude_algo_mock.freeze.assert_called_once()
-        assert scheduler.current_sparsity_level == ref_sparsity_levels[4]
+        assert scheduler.sparsity_level == ref_sparsity_levels[4]
```

Comment:
With schedulers, we've got two major kinds of levels - the target level (of either sparsity or pruning) and the current level. When I see `sparsity_level` or `pruning_level`, it is not immediately obvious to me based on the naming which level is meant, while it *should* be immediately obvious since the mixup between the two types of levels may lead to subtle, hard to debug errors. You should keep both `current` and `target` portions in the level-related attribute name in all affected schedulers.

Reply:
I agree with you. I fixed it.